translate/tools/tests.py

```diff
@@ -0,0 +1,34 @@
+import subprocess
+
+import pytest
+
+from syrupy.assertion import SnapshotAssertion
+
+
+po_files = """
+tests/cli/data/test_pocount_po_csv/one.po
+tests/cli/data/test_pocount_po_file/one.po
+tests/cli/data/test_pocount_po_fuzzy/one.po
+""".split()
```

Comment:
Shouldn't this be declared as an array, instead of a set of strings that is split into an array?